crates/types/Cargo.toml

```diff
@@ -27,6 +27,7 @@ derive_more = { version = "0.99" }
 fuel-vm-private = { workspace = true, default-features = false, features = [
   "alloc",
 ] }
+hex = { version = "0.4", default-features = false }
```

Comment:
Can you not use the `hex` from the workspace?

Reply:
Apparently I can. I had some issues with `default-features` but now it's fine.